ATTRIBUTION.md

```diff
@@ -17,6 +17,108 @@ This package contains a variety of materials, including data sets and related ma
 * Location: ***prereise/gather/data/ba_to_county.txt***
 
 
+#### US County Map Shape Files
+#### Source
+* Name: US County map
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: US county map
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/us-county-boundaries/explore?location=39.896444%2C-101.848119%2C19.58
+#### Destination
+* Modifications to source files(s): None
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Retail Service Territory 
+#### Source
+* Name: Electric Retail Service Territory 
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: electric retail service territory
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/private-non-retail-shipping-facilities/about
+#### Destination
+* Modifications to source files(s): none
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Subregions of NYISO
+#### Source
+* Name: subregions of NYISO
+* Author: ArcGIS
+* Description: subregions of NYISO
+* Source: https://www.arcgis.com/
+* Exact source location: https://www.arcgis.com/home/item.html?id=6fd1de467b134f47a607721f23a69f0c
+#### Destination
+* Modifications to source files(s): None for A-F, they are directly downloaded by the program; G_I are made by grouping and splitting county shapes
+* Location: ***prereise/gather/data/remap_ba_area/nyiso2/***
+
+#### CAISO
+#### Source
+* Name: subregions of CAISO
+* Author: CAISO
+* Description: the subregions of CAISO
+* Source: https://www.caiso.com/
+* Exact source location: https://www.caiso.com/Documents/ISO_ContingencyModelingProposal-SCE_Presentation.pdf
+#### Destination
+* Modifications to source files(s): select county shapes from the image map
+* Location: ***prereise/gather/data/remap_ba_area/CA_balancing_authorities/***
+
+#### PJM
+#### Source
+* Name: subregions of PJM
+* Author: PJM
+* Description: the subregions of PJM
+* Source: PJM
+* Exact source location:  https://www.monitoringanalytics.com/reports/PJM_State_of_the_Market/2018/2018-som-pjm-volume2-appendix.pdf
+#### Destination
+* Modifications to source files(s): The shape files are made from the image.
+* Location: ***prereise/gather/data/remap_ba_area/pjm6.zip***
+
+#### ISONE
+#### Source
+* Name: subregions of ISONE
+* Author: ISONE
+* Description: the subregions of ISONE
+* Source: ISONE
+* Exact source location: https://www.iso-ne.com/isoexpress/web/charts
+#### Destination
+* Modifications to source files(s): the state maps are used
+* Location: ***prereise/gather/data/remap_ba_area/isone.zip***
```

Comment:
If we're just using the state boundaries for ISONE, do we need another input file from ISONE, or can we use state boundaries from the census bureau? (or collect all of the counties within each state from the existing county file)

Reply:
It makes sense to use the existing county map rather than extra shape files.  I made the change and pushed the codes.